Index: EduTest/constructor/static/constructor/questionStyle.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EduTest/constructor/static/constructor/questionStyle.css b/EduTest/constructor/static/constructor/questionStyle.css
new file mode 100644
--- /dev/null	(date 1720290005236)
+++ b/EduTest/constructor/static/constructor/questionStyle.css	(date 1720290005236)
@@ -0,0 +1,159 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #F9F7F7;
+    color: #333;
+}
+
+.container {
+    max-width: 1300px;
+    margin: 0 auto;
+    padding: 20px;
+}
+
+.header {
+    background-color: #3F72AF;
+    color: #ffffff;
+    padding: 10px 20px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    z-index: 1000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 70px;
+}
+
+.header-content {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    margin-left: 30px;
+}
+
+.logo-title {
+    display: flex;
+    align-items: center;
+}
+
+.header-logo {
+    width: 50px;
+    height: auto;
+    margin-right: 5px;
+}
+
+ .header-text {
+     font-size: 27px;
+     font-weight: 700;
+     color:#fff;
+ }
+ .header-text:hover{
+     color: #fff;
+     text-decoration: none;
+ }
+
+.user-links {
+    display: flex;
+    align-items: center;
+    margin-left: -840px;
+}
+
+.user-links a {
+    color: #ffffff;
+    font-weight: 700;
+    text-decoration: none;
+    font-size: 22px;
+}
+
+.user-links a:hover {
+    color: #DBE2EF;
+}
+
+.user-avatar {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    position: absolute;
+    transform: translateY(-50%);
+    right: 100px;
+}
+
+.content {
+    padding-top: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: calc(100vh - 70px);
+    box-sizing: border-box;
+}
+
+.test-form {
+    max-width: 700px;
+    width: 100%;
+    margin: 40px auto 60px;
+    padding: 20px;
+
+}
+
+.test-form h2 {
+    margin-bottom: 20px;
+    color: #3F72AF;
+    font-weight: normal;
+}
+
+.test-form .form-group {
+    margin-bottom: 15px;
+}
+
+.test-form label {
+    font-size: 16px;
+    display: block;
+    margin-bottom: 5px;
+    color: #333;
+}
+
+.test-form input[type="text"],
+.test-form select,
+.test-form input[type="number"],
+.test-form textarea {
+    padding: 6px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+    width: 100%;
+    margin-bottom: 10px;
+}
+
+.test-form textarea {
+    resize: vertical;
+}
+
+.buttons {
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 20px;
+}
+
+.submit-btn-outline-primary {
+    width: auto;
+    border: 2px solid #007bff; /* Dashed border for outline */
+    color: #007bff;
+    border-radius: 4px;
+    padding: 10px;
+    text-align: center;
+    cursor: pointer;
+    white-space: nowrap;
+    background-color: transparent; /
+    margin-left: 10px;
+}
+.submit-btn-outline-primary:hover {
+    background-color: #007bff;
+    color: #ffffff;
+}
+
+.empty-field {
+    border-color: red;
+}
Index: EduTest/constructor/Templates/constructor/Constructor.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Title</title>\r\n</head>\r\n<body>\r\n<form method=\"post\">\r\n    {% csrf_token %}\r\n    {{ form.as_p }}\r\n    <button type=\"submit\">Далее</button>\r\n</form>\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EduTest/constructor/Templates/constructor/Constructor.html b/EduTest/constructor/Templates/constructor/Constructor.html
--- a/EduTest/constructor/Templates/constructor/Constructor.html	(revision c893a6c9a8a8add9d3cf702a63fb94d64e64b2fa)
+++ b/EduTest/constructor/Templates/constructor/Constructor.html	(date 1720286090780)
@@ -1,14 +1,46 @@
+{% load static %}
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EduTest</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap">
+    <link rel="shortcut icon" href="{% static 'constructor/mini-icon.png' %}" type="image/x-icon">
+    <link rel="stylesheet" href="{% static 'constructor/Constructor.css' %}">
 </head>
 <body>
-<form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <button type="submit">Далее</button>
-</form>
+<div class="header">
+    <div class="container header-content">
+        <div class="logo-title">
+            <img src="{% static 'constructor/header.png' %}" alt="Лого" class="header-logo">
+            <a href="#">
+                <span class="header-text">EduTest</span>
+            </a>
+        </div>
+        <div class="user-links">
+            <a href="#">Тесты</a>
+        </div>
+        <a href="#">
+            <img id="userAvatar" src="{% static 'constructor/profileIcon-image.png' %}" alt="User Avatar"
+                 class="user-avatar">
+        </a>
+    </div>
+</div>
+<div class="content container">
+    <form class="test-form" method="post">
+        <div class="form-group">
+            {% csrf_token %}
+            {% for field in form %}
+                <div class="form-field">
+                    <label for="{{ field.id_for_label }}">{{ field.label }}</label>
+                    {{ field }}
+                </div>
+            {% endfor %}
+            <button type="submit" class="btn btn-outline-primary">Далее</button>
+        </div>
+    </form>
+</div>
 </body>
 </html>
\ No newline at end of file
Index: EduTest/constructor/Templates/constructor/question.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Title</title>\r\n</head>\r\n<body>\r\n<form method=\"post\">\r\n    {% csrf_token %}\r\n    <h2>Question</h2>\r\n    {{ question_form.as_p }}\r\n    <h2>Answers</h2>\r\n    {{ answer_formset.management_form }}\r\n    {% for form in answer_formset %}\r\n        {{ form.as_p }}\r\n    {% endfor %}\r\n    <button type=\"submit\">Добавить следующий вопрос</button>\r\n</form>\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EduTest/constructor/Templates/constructor/question.html b/EduTest/constructor/Templates/constructor/question.html
--- a/EduTest/constructor/Templates/constructor/question.html	(revision c893a6c9a8a8add9d3cf702a63fb94d64e64b2fa)
+++ b/EduTest/constructor/Templates/constructor/question.html	(date 1720289668308)
@@ -1,20 +1,49 @@
+{% load static %}
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EduTest</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap">
+    <link rel="shortcut icon" href="{% static 'constructor/mini-icon.png' %}" type="image/x-icon">
+    <link rel="stylesheet" href="{% static 'constructor/questionStyle.css' %}">
 </head>
 <body>
-<form method="post">
-    {% csrf_token %}
-    <h2>Question</h2>
-    {{ question_form.as_p }}
-    <h2>Answers</h2>
-    {{ answer_formset.management_form }}
-    {% for form in answer_formset %}
-        {{ form.as_p }}
-    {% endfor %}
-    <button type="submit">Добавить следующий вопрос</button>
-</form>
+<div class="header">
+    <div class="container header-content">
+        <div class="logo-title">
+            <img src="{% static 'constructor/header.png' %}" alt="Лого" class="header-logo">
+            <a href="#">
+                <span class="header-text">EduTest</span>
+            </a>
+        </div>
+        <div class="user-links">
+            <a href="#">Тесты</a>
+        </div>
+        <a href="#">
+            <img id="userAvatar" src="{% static 'constructor/profileIcon-image.png' %}" alt="User Avatar"
+                 class="user-avatar">
+        </a>
+    </div>
+</div>
+<div class="content">
+    <form method="post" class="test-form">
+        {% csrf_token %}
+        <div class="form-group">
+            {{ question_form.as_p }}
+        </div>
+        {{ answer_formset.management_form }}
+        {% for form in answer_formset %}
+        <div class="form-group">
+            {{ form.as_p }}
+        </div>
+        {% endfor %}
+        <div class="buttons">
+            <button type="submit" class="btn submit-btn-outline-primary">Добавить следующий вопрос</button>
+        </div>
+    </form>
+</div>
 </body>
-</html>
\ No newline at end of file
+</html>
Index: EduTest/constructor/static/constructor/Constructor.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EduTest/constructor/static/constructor/Constructor.css b/EduTest/constructor/static/constructor/Constructor.css
new file mode 100644
--- /dev/null	(date 1720290005250)
+++ b/EduTest/constructor/static/constructor/Constructor.css	(date 1720290005250)
@@ -0,0 +1,156 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #F9F7F7;
+    color: #333;
+}
+
+.container {
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 20px;
+}
+
+.header {
+    background-color: #3F72AF;
+    color: #ffffff;
+    padding: 10px 20px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    z-index: 1000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 70px;
+}
+
+.header-content {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    margin-left: 30px;
+}
+
+.logo-title {
+    display: flex;
+    align-items: center;
+}
+
+.header-logo {
+    width: 50px;
+    height: auto;
+    margin-right: 5px;
+}
+
+ .header-text {
+     font-size: 27px;
+     font-weight: 700;
+     color:#fff;
+ }
+ .header-text:hover{
+     color: #fff;
+     text-decoration: none;
+ }
+
+.user-links {
+    display: flex;
+    align-items: center;
+    margin-left: -740px;
+}
+
+.user-links a {
+    color: #ffffff;
+    font-weight: 700;
+    text-decoration: none;
+    font-size: 22px;
+}
+
+.user-links a:hover {
+    color: #DBE2EF;
+}
+
+.user-avatar {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    position: absolute;
+    transform: translateY(-50%);
+    right: 100px;
+}
+
+.content {
+    padding-top: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 80vh;
+}
+.test-form{
+    margin-bottom: 20px;
+}
+.test-form.form-group {
+    margin-bottom: 20px;
+}
+
+.test-form label {
+    display: block;
+    margin-bottom: 10px;
+    font-weight: lighter;
+}
+
+.test-form input {
+    width: 500px;
+    padding: 10px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    height: 35px;
+    font-size: 16px;
+    margin-bottom: 20px ;
+}
+
+.test-form input:focus {
+    border-color: #007bff;
+    box-shadow: 0 0 10px rgba(0, 123, 255, 0.5);
+}
+
+
+.test-form button[type="submit"] {
+    color: #007bff;
+    border-color: #007bff;
+    background-color: #ffffff;
+    padding: 8px 20px;
+    border-radius: 4px;
+    cursor: pointer;
+
+}
+
+.test-form button[type="submit"]:hover {
+    background-color: #007bff;
+    color: #ffffff;
+}
+.test-form select {
+    width: 100%;
+    padding: 8px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 16px;
+    background-color: #ffffff;
+    color: #333;
+    overflow: visible;
+    white-space: normal;
+    margin-bottom: 20px;
+}
+
+.btn-outline-primary {
+    color: #007bff;
+    border-color: #007bff;
+}
+
+.btn-outline-primary:hover {
+    background-color: #007bff;
+    color: #ffffff;
+}
+
